lib/Doctrine/DBAL/Psr11ConnectionRegistry.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL;
```

Comment:
should we put that class at the root, or in a `Registry` subnamespace, to allow grouping implementations if needed ?

Reply:
Alternative suggestion for a namespace:
```suggestion
namespace Doctrine\DBAL\Connection;
```

Reply:
Then I think I would go for `Registry`. Otherwise I find it confusing that the `Connection` itself is not in the `Connection` namespace :smile: 

Or?

Reply:
Changed it to `Doctrine\DBAL\Registry` for now